cli/ioctl/cmd/action/action.go

```diff
@@ -166,3 +148,43 @@ func sendAction(elp action.Envelope, readOnly bool) (string, error) {
 		"Wait for several seconds and query this action by hash:\n" +
 		hex.EncodeToString(shash[:]), nil
 }
+
+func readAction(exec *action.Execution) (string, error) {
+	fmt.Printf("Enter password #%s:\n", signer)
+	bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		log.L().Error("failed to get password", zap.Error(err))
+		return "", err
+	}
+	prvKey, err := account.KsAccountToPrivateKey(signer, string(bytePassword))
+	if err != nil {
+		return "", err
+	}
+	defer prvKey.Zero()
+	addr, err := address.FromBytes(prvKey.PublicKey().Hash())
```

Comment:
We can let user pass the address or alias in for this command

Reply:
yes, can pass in address from above